### PR TITLE
Merge 1.5 into master

### PR DESCRIPTION
**Pull request by damien-carcel**
**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Yes
| Added Behats                      | Yes
| Changelog updated                 | Yes
| Review and 2 GTM                  | Pending
| Micro Demo to the PO (Story only) | N/A
| Migration script                  | No
| Tech Doc                          | No

### CODE REVIEW

File: src/Pim/Bundle/ConnectorBundle/Resources/translations/messages.pt.yml [hunk 1]
```diff
@@ -76,87 +129,3 @@ pim_connector:
...
-    validation.label: Validação do ficheiro
-  csv_group_import:
-    label: Importar grupo para o CSV
-    validation.label: Validação do ficheiro
-    import.label: Importar grupo
-  csv_variant_group_import:
-    label: Importar variante do grupo para o CSV
-    validation.label: Validação do ficheiro
-    import.label: Importar variante do grupo
-  xlsx_category_import:
```
Comment (skeleton) [inline src/Pim/Bundle/ConnectorBundle/Resources/translations/messages.pt.yml]: normal to remove all xlsx_* keys ?

File: src/Pim/Bundle/EnrichBundle/Resources/config/controllers.yml [hunk 2]
```diff
@@ -30,6 +30,7 @@ parameters:
     pim_enrich.controller.rest.locale.class:                  Pim\Bundle\EnrichBundle\Controller\Rest\LocaleController
     pim_enrich.controller.rest.measures.class:                Pim\Bundle\EnrichBundle\Controller\Rest\MeasuresController
     pim_enrich.controller.rest.media.class:                   Pim\Bundle\EnrichBundle\Controller\Rest\MediaController
+    pim_enrich.controller.rest.navigation_history.class: Pim\Bundle\EnrichBundle\Controller\Rest\NavigationHistoryController
```
Comment (skeleton) [inline src/Pim/Bundle/EnrichBundle/Resources/config/controllers.yml]: indent please

**Comment by skeleton** [comment]
gtm once fixed

File: src/Pim/Bundle/ConnectorBundle/Resources/translations/messages.pt.yml (hunk 1 above)
Comment (damien-carcel) [inline src/Pim/Bundle/ConnectorBundle/Resources/translations/messages.pt.yml]: clearly not…

**Pull request merged**